frontend/packages/modelServing/src/components/global/GlobalDeploymentsTable.tsx

```diff
@@ -1,5 +1,126 @@
 import React from 'react';
+import ModelServingToolbar from '@odh-dashboard/internal/pages/modelServing/screens/global/ModelServingToolbar';
+import {
+  initialModelServingFilterData,
+  type ModelServingFilterDataType,
+} from '@odh-dashboard/internal/pages/modelServing/screens/global/const';
+import { getDisplayNameFromK8sResource } from '@odh-dashboard/internal/concepts/k8s/utils';
+import DashboardEmptyTableView from '@odh-dashboard/internal/concepts/dashboard/DashboardEmptyTableView';
+import { namespaceToProjectDisplayName } from '@odh-dashboard/internal/concepts/projects/utils';
+import { ProjectsContext } from '@odh-dashboard/internal/concepts/projects/ProjectsContext';
+import { useExtensions, useResolvedExtensions } from '@odh-dashboard/plugin-core';
+import type { ProjectKind } from '@odh-dashboard/internal/k8sTypes';
+import { Label } from '@patternfly/react-core';
+import DeploymentsTable from '../deployments/DeploymentsTable';
+import {
+  isModelServingDeploymentsTableExtension,
+  isModelServingPlatformExtension,
+  type Deployment,
+  type DeploymentsTableColumn,
+} from '../../../extension-points';
 
-const GlobalDeploymentsTable: React.FC = () => <>Global Model Deployments Table</>;
+const ProjectCell: React.FC<{ deployment: Deployment; projects: ProjectKind[] }> = ({
+  deployment,
+  projects,
+}) => {
+  const platformLabel = useExtensions(isModelServingPlatformExtension);
+  const platform = platformLabel.find((p) => p.properties.id === deployment.modelServingPlatformId);
+  return (
+    <>
+      {namespaceToProjectDisplayName(deployment.model.metadata.namespace, projects)}
```

Comment:
I think you need to add a space here at the end to make it look like we do without the plugin